doc/releases/changelog-dev.md

```diff
@@ -925,6 +925,10 @@
 
 <h3>Deprecations</h3>
 
+* The `default.tensor` device from the beta folder has not been maintained in 
```

Comment:
Just to make sure, the same with `default.tensor.tf`, right?

Reply:
Well, it is kind of a subcase of `default.tensor` so I think it is included here?